docs/sidebars.js

```diff
@@ -99,7 +99,6 @@ const sidebars = {
           "items": [
             "reference/command-line/acorn",
             "reference/command-line/acorn_all",
-            "reference/command-line/acorn_app",
```

Comment:
I wonder if we should have redirects for renames :thinking: 
Potentially some blog post is linking to it (though I hope that none links to generated content).

Reply:
Good call, I added a redirect (and tested it to make sure it works). I modified the docs generator a little bit so that it won't clobber the `.mdx` file I created.